packages/accounts-controller/src/AccountsController.test.ts

```diff
@@ -1107,6 +1150,50 @@ describe('AccountsController', () => {
           mockAccountWithoutLastSelected,
         );
       });
+
+      it('publishes accountRemoved event', async () => {
+        const messenger = buildMessenger();
+        const messengerSpy = jest.spyOn(messenger, 'publish');
+        mockUUID
+          .mockReturnValueOnce('mock-id') // call to check if its a new account
+          .mockReturnValueOnce('mock-id2') // call to check if its a new account
+          .mockReturnValueOnce('mock-id2'); // call to add account
```

Comment:
Would be great to use `mockAccount*.id` here (in case we change id some days)
```suggestion
          .mockReturnValueOnce(mockAccount.id) // call to check if its a new account
          .mockReturnValueOnce(mockAccount2.id) // call to check if its a new account
          .mockReturnValueOnce(mockAccount2.id); // call to add account
```